tortoise/backends/sqlite/executor.py

```diff
@@ -76,6 +77,11 @@ def to_db_time(
     return None
 
 
+# Converts Decimal to string for sqlite in cases where it's hard to know the
+# related field, e.g. in raw queries, math or annotations.
+sqlite3.register_adapter(Decimal, str)
```

Comment:
Django uses the same approach https://github.com/django/django/blob/857b1048d53ebf5fc5581c110e85c212b81ca83a/django/db/backends/sqlite3/base.py#L55